src/main/java/com/iyzipay/model/CardManagementPageInitialize.java

```diff
@@ -0,0 +1,47 @@
+package com.iyzipay.model;
+
+import com.iyzipay.HttpClient;
+import com.iyzipay.IyzipayResource;
+import com.iyzipay.Options;
+import com.iyzipay.request.CreateCardManagementPageInitializeRequest;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+public class CardManagementPageInitialize extends IyzipayResource {
+
+    private String externalId;
+    private String token;
+    private String cardPageUrl;
+
+    public static CardManagementPageInitialize create(CreateCardManagementPageInitializeRequest request, Options options) {
+        return HttpClient.create().post(options.getBaseUrl() + "/api/v1/forms",
```

Comment:
Base url will be set like this: "https://api.iyzipay.com/api/v1/forms" ? If yes "api" words duplicated and we can not understand which form it is. You should write card management word to url. It should be https://api.iyzipay.com/v1/card-management/forms or https://cardapi.iyzipay.com/v1/forms.

Reply:
hi @yyenigun , baseUrl will be set like as in live environment "[https://api.iyzipay.com/api/v1/card-management/forms](https://api.iyzipay.com/api/v1/card-management/forms)"

Reply:
@AlicanAkkus url should be like this: "https://api.iyzipay.com/v1/card-management/forms" and you should also change it in the code: options.getBaseUrl() + "/v1/card-management/forms"

Reply:
@yyenigun yo're right. I've changed it in the code.